tasks/run-local.sh

```diff
@@ -12,6 +12,9 @@ SECRETS=$DATADIR/secrets
 IMAGES=$DATADIR/images
 IMAGE_PORT=${IMAGE_PORT:-8080}
 S3_PORT=${S3_PORT:-9000}
+S3_URL_POD=http://localhost.localdomain:9000
```

Comment:
This could use a comment.  It took me a fair bit of thinking before I realised what the point of having two of these was.

Reply:
Ack, added a comment.